_overviews/hello-scala/scala-features.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: multipage-overview
+title: Scala Features
+description: TODO
+partof: hello_scala
+overview-name: Hello, Scala
+discourse: true
+num: 4
+outof: 54
+previous-page: getting-started
+next-page: hello-world-1
+---
+
+
+The name *Scala* comes from the word *scalable*, and true to that name, it’s used to power the busiest websites in the world, including Twitter, Netflix, Tumblr, LinkedIn, Foursquare, and many more.
+
+Here are a few more nuggets about Scala:
+
+- It’s a modern programming language created by [Martin Odersky](https://twitter.com/odersky?lang=en) (the father of `javac`), and influenced by Java, Ruby, Smalltalk, ML, Haskell, Erlang, and others.
+- It’s a high-level language.
+- It’s statically typed.
+- It has a sophisticated type inference system.
+- It’s syntax is concise but still readable — we call it *expressive*.
+- It’s a pure object-oriented programming (OOP) language. Every variable is an object, and every “operator” is a method.
+- It’s also a functional programming (FP) language, so functions are also variables, and you can pass them into other functions. You can write your code using OOP, FP, or combine them in a hybrid style.
+- Scala source code compiles to “.class” files that run on the JVM.
+- Scala also works extremely well with the thousands of Java libraries that have been developed over the years.
+- A great thing about Scala is that you can be productive with it on Day 1, but it’s also a deep language, so as you go along you’ll keep learning, and finding newer, better ways to write code. It’s said that Scala will change the way you think about programming (and that’s a good thing).
```

Comment:
I think it's a nice literary concession. Backing up this claim with data it's difficult and given that a book synthesizes the communication between an author and the readers, I don't mind reading this kind of praise in a book about learning Scala. It's motivational and I think people will feel they wanna learn more.

Reply:
I originally wrote this as a third-party, so I didn’t mind making statements like this, especially because I know it to be true in my own case. There were so many things I didn’t think about when I wrote Java/OOP code, and now that I’ve been working with Scala (and FP), I know it’s changed how I think about writing code.

If you would like to call this “Scala Book” and we need to be more conservative, that’s fine. Just let me know what you all think.

Reply:
> I don't mind reading this kind of praise in a book about learning Scala. It's motivational and I think people will feel they wanna learn more.

At least for me, reading such a phrase in book to learn something, I would rather become wary than motivated.

> If you would like to call this “Scala Book” and we need to be more conservative, that’s fine. Just let me know what you all think.

Maybe, we can just change for the following?

```suggestion
- A great thing about Scala is that you can be productive with it on Day 1, but it’s also a deep language, so as you go along you’ll keep learning, and finding newer, better ways to write code. Some people say that Scala will change the way you think about programming (and that’s a good thing).
```